env/elasticsearch/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #https://www.docker.elastic.co/
-FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.2
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.5.2
```

Comment:
Could you also update the README file with this version. The testing version is mentioned there.

Reply:
yep sorry my fault :)

Reply:
@ruflin done!